.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
If this is not intentional, remove this file from pushing, add to gitignore and delete it